src/test/rules/sortImportsRuleTests.ts

```diff
@@ -0,0 +1,189 @@
+  {
+    code: `import a from 'foo';
+        import A from 'bar';`,
+    errors: expecting([{ message: formatString(ALPHA_ORDER_ERROR, 'A', 'a'), line: 1, startColumn: 8, endColumn: 28 }])
+  },
+  {
+    code: `import b from 'foo';
+        import a from 'bar';`,
+    errors: expecting([{ message: formatString(ALPHA_ORDER_ERROR, 'a', 'b'), line: 1, startColumn: 8, endColumn: 28 }])
+  },
+  {
+    code: `import {b, c} from 'foo';
+        import {a, d} from 'bar';`,
+    errors: expecting([{ message: formatString(ALPHA_ORDER_ERROR, 'a', 'b'), line: 1, startColumn: 8, endColumn: 33 }])
+  },
+  {
+    code: `import * as foo from 'foo'
+        import * as bar from 'bar';`,
+    errors: expecting([{ message: formatString(ALPHA_ORDER_ERROR, 'bar', 'foo'), line: 1, startColumn: 8, endColumn: 35 }])
+  },
+  {
+    code: `import a from 'foo';
+        import {b, c} from 'bar';`,
+    errors: expecting([{ message: formatString(TYPE_ORDER_ERROR, 'Multiple', 'Single'), line: 1, startColumn: 8, endColumn: 33 }])
+  },
+  {
+    code: `import a from 'foo';
+        import * as b from 'bar';`,
+    errors: expecting([{ message: formatString(TYPE_ORDER_ERROR, 'All', 'Single'), line: 1, startColumn: 8, endColumn: 33 }])
+  },
+  {
+    code: `import a from 'foo';
+        import 'bar';`,
+    errors: expecting([{ message: formatString(TYPE_ORDER_ERROR, 'None', 'Single'), line: 1, startColumn: 8, endColumn: 21 }])
+  },
+  {
+    code:
+    `import b from 'bar';
+        import * as a from 'foo';`,
+    options: [{ 'member-syntax-sort-order': ['all', 'single', 'multiple', 'none', 'alias'] }],
+    errors: expecting([{ message: formatString(TYPE_ORDER_ERROR, 'All', 'Single'), line: 1, startColumn: 8, endColumn: 33 }])
+  },
+  {
+    code: `import {b, a, d, c} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 19 }])
+  },
+  {
+    code: `import {a, B, c, D} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 19 }])
+  },
+  {
+    code: `import {zzzzz, /* comment */ aaaaa} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 35 }])
+  },
+  {
+    code: `import {zzzzz /* comment */, aaaaa} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 35 }])
+  },
+  {
+    code: `import {/* comment */ zzzzz, aaaaa} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 35 }])
+  },
+  {
+    code: `import {zzzzz, aaaaa /* comment */} from 'foo';`,
+    errors: expecting([{ message: MEMBER_SORT_ERROR, line: 0, startColumn: 7, endColumn: 35 }])
+  },
+  {
+    code: `
+              import {
+                boop,
+                foo,
+                zoo,
+                baz as qux,
+                bar,
+                beep
+              } from 'foo';
+            `,
+    errors: [{
```

Comment:
I think it's because the alias import syntax is unique to Typescript, the ES6 import declaration doesn't seem to have it:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import